_episodes/04-sharing-environments.md

```diff
@@ -135,27 +144,31 @@ originally created `machine-learning-env` yet from the output of the `conda env
 we see that these five packages result in an environment with roughly 80 dependencies!
 
 To export this list into an environment.yml file, you can use `--file` option to directly save the
-resulting YAML environment into a file.
+resulting YAML environment into a file. If the target for `--file` exists it will be over-written so make sure your
+filename is unique. So that we do not over-write `environment.yaml` we save the output to `machine-learning-env.yaml`
+instead and add it to
 
 ~~~
-$ conda env export --name machine-learning-env --file environment.yml
+$ conda env export --name machine-learning-env --file machine-learning-env.yml
+$ git init

```

Comment:
I think it's good to include this much git.

Reply:
Cool, enough but no more.